src/lib/components/forms/form/form-components/text-area/text-area.component.spec.ts

```diff
@@ -81,6 +81,19 @@ describe('TextAreaComponent', () => {
         expect(await input.isDisabled()).toBe(true);
     });
 
+    it('should only accept #maxLength amount of characters', async () => {
+        const maxLength: number = 10;
+        const validString: string = new Array(maxLength).fill('x').join('');
+        const invalidString: string = new Array(maxLength + 1).fill('x').join('');
+        component.maxLength = maxLength;
+
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+        await input.setValue(validString);
+        expect(await formField.isControlValid()).toBe(true);
+        await input.setValue(invalidString);
+        expect(await formField.isControlValid()).toBe(false);
+    });
```

Comment:
Am liebsten auch noch so einen Test für minLength.

Kannst du ggf. in dem Zuge auch noch prüfen ob die Fehlermeldung angezeigt wird?
Dafür müsstest du das input natürlich wieder verlassen (`input.blur()`)